diff -up dialog-1.1-20080819/guage.c.orig dialog-1.1-20080819/guage.c
--- dialog-1.1-20080819/guage.c.orig	2007-02-24 01:06:25.000000000 +0100
+++ dialog-1.1-20080819/guage.c	2015-03-27 15:14:54.864843287 +0100
@@ -228,9 +228,6 @@ dialog_gauge(const char *title,
     nodelay(stdscr, FALSE);
 #endif
 
-    fclose(dialog_state.pipe_input);
-    dialog_state.pipe_input = 0;
-
     curs_set(1);
     dlg_del_window(dialog);
     if (prompt != prompt_buf)
