diff -up dialog-1.1-20120706/makefile.in.libs dialog-1.1-20120706/makefile.in
--- dialog-1.1-20120706/makefile.in.libs	2012-02-16 01:35:37.000000000 +0100
+++ dialog-1.1-20120706/makefile.in	2012-11-13 19:53:14.193243663 +0100
@@ -178,11 +178,11 @@ dialog$o \
 $(OBJECTS) : $(srcdir)/dialog.h $(srcdir)/dlg_keys.h dlg_config.h VERSION
 
 $(LIB) : $(LIB_OBJECT)
-	$(LIBTOOL_CREATE) $(LIB) $(LIB_OBJECT)
+	$(LIBTOOL_CREATE) $(LIB) $(LIB_OBJECT) $(LIBS)
 	$(RANLIB) $@
 
 $(PROG)$x : $(LIB) dialog$o @INTLDIR_MAKE@ @INTLLIBS@
-	$(LINK) -o $@ dialog$o -L. -l@PACKAGE@ $(LDFLAGS) $(LIBS)
+	$(LINK) -o $@ dialog$o -L. -l@PACKAGE@ $(LDFLAGS)
 
 clean	\
 distclean \
